packages/airnode-deployer/src/cli/commands.ts

```diff
@@ -68,8 +64,14 @@ export async function deploy(configPath: string, secretsPath: string, receiptFil
   }
 
   const output = goDeployAirnode.data;
-  if (output.httpGatewayUrl) logger.info(`HTTP gateway URL: ${output.httpGatewayUrl}`);
-  if (output.httpSignedDataGatewayUrl) logger.info(`HTTP signed data gateway URL: ${output.httpSignedDataGatewayUrl}`);
+  if (output.httpGatewayUrl) {
+    logger.setSecret(output.httpGatewayUrl);
+    logger.info(`HTTP gateway URL: ${output.httpGatewayUrl}`);
+  }
+  if (output.httpSignedDataGatewayUrl) {
+    logger.setSecret(output.httpSignedDataGatewayUrl);
+    logger.info(`HTTP signed data gateway URL: ${output.httpSignedDataGatewayUrl}`);
+  }
```

Comment:
I went with this option to set secret values to be sanitized from logs. The deployer command options are passed down multiple levels and I would have needed to pass the list of secret values along for them to reach the logger call. So this seemed simpler.